### DISCUSSION TITLE
Welcome to Video Hub App Discussions!

### DISCUSSION THREAD

**Discussion by whyboris** [whyboris/Video-Hub-App#604] [General]
## 👋 Welcome!

  I'm trying out _Discussions_ as a place to connect with others who use *Video Hub App*. I hope that you:
  * Ask questions you’re wondering about
  * Share ideas
  * Engage with other community members
  * Welcome others and are open-minded. Remember that this is a community we build together 💪

If you'd like, comment below however you'd like:
- an introduction of yourself 
- tell us about how you use *Video Hub App*
- anything you think is relevant 👋

**Comment by ToooOldToCode** [comment]
Hi, I was wondering if there is an option to add multiple tags at once to a video (many tags seperated by comma), instead of adding them one by one.
i used to do this by adding all the tags in the .bak file, but it looks like this is not the case anymore.

**Reply by ToooOldToCode** [reply]
i was reading some features on the website and i saw this:
_ Batch tagging
can you show me how exactly i can use this please, it sounds like the thing im looking for.

**Reply by whyboris** [reply]
Hello. The `.bak` file is meant as a backup - only useful if the original `.vha2` file gets corrupt. It is never automatically used either - so editing it doesn't affect your hub unless you rename `.vha2.bak` into `.vha2`

You can still go and manually edit the `.vha2` file as you'd like (just please, back it up first in case your edits corrupt the formatting - which is just a JSON format btw).

*Batch tagging* is a feature in the _tag tray_ (the tray that appears on the bottom - plese enable in settings if you accidentally disabled it). In the _tag tray_ on the bottom-left is "batch" mode which lets you select many videos at once and then you can click on a tag to add it to all selected videos.

**Reply by ToooOldToCode** [reply]
thank you so much for the quick answer, and i know that you can add a tag to multiple videos at once, but what about adding multiple tags to one video?
lol now i see what i was doing wrong i was editing the .bak file not the .vha2, i thought this last one is some sort of executable my bad. no wonder why no changes were applied when i launched the app.
thanks again for your hard work, and your great app.
keep up the good work

**Reply by whyboris** [reply]
I'm unsure how a UI (user interface) for "add multiple tags to one video" would even work - you'd have to pre-define a group of tags, store it somehow, and then apply the group to a video. But then, what if the user wants to change the group (add/remove/edit) - it's all too much of a hassle for no (to me) visible benefit over just clicking on several tags.

ps - I hope you know there is _auto-complete_ when you type in a tag -- it shows up in blue and you can press the `TAB` key on your keyboard to auto-complete and add the tag 🚀

**Comment by okumam** [comment]
It would be great if there was a way to sort the automatically created tags alphabetically instead of what it is right now (assuming frequency?).  I have many videos that I used keywords in the filenames to locate before I found video hub app.  So for the kids, Jack and Jane, the video file name has their names, event, month, and so on.  Jack_Birthday_2021_party_family.mpg for example.  It's great that VHA can generate filename tags automatically using these words but then the list of the tags is a jumble.  It's hard to find Jack or Birthday in that list- you have to read the whole list over and over again.  I really wish you could have an option where the tags are sorted alphabetically.

What do you think?

**Reply by whyboris** [reply]
Thank you for the feedback! Currently the automatically-generated tags are in the same order as filename, so `Jack_Birthday_2021_party_family` should become `jack`, `birthday`, `2021`, `party`, `family`.

I hadn't thought that alphabetizing the tags in the auto-generated tags might be useful - now that I think of it, it seems like a good idea. 

Perhaps it would be best to leave functionality as is, but add an additional setting button that will toggle the tags in alphabetical order (as you recommend) 🤔 

I've added an _Issue_ for this task: #713 🤝
